### PR TITLE
Main Testing 2

### PR DESCRIPTION
**Pull request by realjoshuau**

### CODE REVIEW

**Comment by ChaoticChaosTheGreat** [comment]
intake stuff merge

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -314,15 +352,18 @@ public static final class IntakeConstants {
         450; // todo: tune | meters probably needs to be tuned
 
     /* Presets */
-    public static final double kIntakeNoteRPM = 1000;
+    public static final double kIntakeNoteRPM = 6000; // 1000
```
Comment (realjoshuau) [inline src/main/java/frc/robot/Constants.java:355]: not used you should use this

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -314,15 +352,18 @@ public static final class IntakeConstants {
...
 
     /* Misc */
     public static final int kCurrentThreshold = 16;
     public static final boolean kUseIntakeMotionMagic = false;
 
     // Time of Flight constants
-    public static final boolean kUseTimeOfFlight = true;
+    public static final boolean kUseTimeOfFlight = false;
+    public static final boolean kUseTimedCurrentSpike = true;
+    public static final double kIntakeCurrentThresholdIntakeSequence = 20;
```
Comment (realjoshuau) [inline src/main/java/frc/robot/Constants.java:365]: nitpick: wordy

File: src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -300,6 +302,18 @@ private void configureShooter() {
       shooter = new Shooter(new ShooterIOTalonFX());
       shooter = new Shooter(new ShooterIOTalonFX());
     }
+    new Trigger(() -> Math.abs(shooter.getShooterRps() - 100) <= 5)
+        .onTrue(
+            new InstantCommand(
+                () -> {
+                  operator.getHID().setRumble(GenericHID.RumbleType.kBothRumble, 100);
+                }));
+    new Trigger(() -> Math.abs(shooter.getShooterRps() - 100) > 5)
```
Comment (realjoshuau) [inline src/main/java/frc/robot/RobotContainer.java:311]: does this work

**Review by realjoshuau — COMMENTED** [review]
no CANdle makes @WilliamT424 very sad

**Pull request merged**